### ISSUE TITLE
Improve user interface

### ISSUE THREAD

**Issue by dielsonsales** [eleybourn/Book-Catalogue#794]
Despite the app being very useful, I think its design is a bit outdated compared to today's Android standards. Because of that, I thought about implementing a Material Design like visual, so I'd like to know if you guys have an opinion about that, since that requires the Google Support Design library for backporting features before Lollipop.

**Comment by dielsonsales** [comment]
Yeah, that library is also deprecated as it's stated in the Github repository.

Okay, my only concern was something like you guys being against any Google libraries being included. I'll start working on it in my free times, so I'll let you know if I have something to show up.

**Comment by Grunthos** [comment]
Unfortunately, I strongly suspect that if you want to use latest design libraries (and latest Android Studio/Gradle), Action Bar Sherlock needs to be removed (which I why I mentioned it).

**Comment by Djiko** [comment]
Hello @dielsonsales  Did you make any progress on that? I'd really like to see the app moving forward this way.

**Comment by Grunthos** [comment]
FWIW, current sources build with API target 28 and ABS is removed.

**Comment by Grunthos** [comment]
A complete UI retrofit is a very large task with very little reward...let me know if you are still interested.

**Comment by dielsonsales** [comment]
I made a simple material design version while trying out Adobe XD, I'll try to post it here tonight.
